src/clusterd/src/lib.rs

```diff
@@ -212,11 +224,10 @@ async fn run(args: Args) -> Result<(), anyhow::Error> {
     mz_alloc::register_metrics_into(&metrics_registry).await;
     mz_metrics::register_metrics_into(&metrics_registry, mz_dyncfgs::all_dyncfgs()).await;
 
-    if let Some(memory_limit) = args.announce_memory_limit {
-        mz_compute::lgalloc::start_limiter(memory_limit, &metrics_registry);
-        mz_compute::memory_limiter::start_limiter(memory_limit, &metrics_registry);
+    if let Some(heap_limit) = args.heap_limit {
+        mz_compute::memory_limiter::start_limiter(heap_limit, &metrics_registry);
     } else {
-        warn!("no memory limit announced; disabling lgalloc limiter");
+        warn!("no heap limit announced; disabling memory limiter");
```

Comment:
Maybe we should change this to be less than `warn`? It spams tests this way, but I also see why we want it to warn...